### PR TITLE
fix: update aws lambda docs for updated UI

### PR DESCRIPTION
**Pull request by scefali**
We recently updated the UI for the AWS Lambda integration here: https://github.com/getsentry/sentry/pull/23926

This PR just updates the instructions to reflect that plus a few minor changes

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
Could you take the screenshot using cmd+shift+4 and press space to capture the window

**Review by EvanPurkhiser — APPROVED** [review]

File: src/docs/product/integrations/aws-lambda/index.mdx [hunk 1]
```diff
@@ -26,25 +26,29 @@ Connect Sentry to your AWS account to automatically add Sentry error and perform
...
 
 <Alert title="Note" level="info">
 
-You need to create the CloudFormation stack only once per account, even when installing the integration in multiple regions. If you are setting up the integration in a second region, verify that the externalId in the form matches the one for the existing CloudFormation stack on the "Parameters" tab.
+You need to create the CloudFormation stack only once per account, even when installing the integration in multiple regions. If you are setting up the integration in a second region, copy the externalId from the existing CloudFormation stack on the "Parameters" tab into the form shown above.
 
 </Alert>
 
-1. Select the Lambda functions to automatically add the Sentry layer and necessary environment variables when you **Finish Setup**. **Note:** You can enable or disable Sentry on Lambda functions after installation.
+8. Select the Lambda functions to automatically add the Sentry layer and necessary environment variables when you **Finish Setup**. **Note:** You can enable or disable Sentry on Lambda functions after installation.
```
Comment (wmak) [inline src/docs/product/integrations/aws-lambda/index.mdx:51]: I don't think this markdown is formatting the number correctly
image

Comment (scefali) [reply src/docs/product/integrations/aws-lambda/index.mdx:51]: Is that from before? Cause this is what I see in the preview build:

Screen Shot 2021-02-18 at 5 41 23 PM

**Comment by scefali** [comment]
> Could you take the screenshot using cmd+shift+4 and press space to capture the window

@EvanPurkhiser Great tip!

File: src/docs/product/integrations/aws-lambda/index.mdx (hunk 1 above)
Comment (wmak) [reply src/docs/product/integrations/aws-lambda/index.mdx:51]: Nevermind, yup. TIL the preview's search switches to docs.sentry.io

**Review by wmak — APPROVED** [review]

**Comment by EvanPurkhiser** [comment]
The drop shadow might be a lot though, you can turn it off if you want https://www.dessol.com/blog/disable-drop-shadow-mac-osx-window-screenshots/

File: src/docs/product/integrations/aws-lambda/index.mdx (hunk 1 above)
Comment (PeloWriter) [reply src/docs/product/integrations/aws-lambda/index.mdx:51]: Yes, that can happen :( - you think you're looking at the preview, when in fact you're in live docs. Only way to confirm what you're seeing is by viewing the URL.

File: src/docs/product/integrations/aws-lambda/index.mdx [hunk 2]
```diff
@@ -26,25 +26,29 @@ Connect Sentry to your AWS account to automatically add Sentry error and perform
...
+1. Now copy your AWS account number, select the region you want to use, and press **Next**.
 
    ![](stack_created.png)
 
 <Alert title="Note" level="info">
 
-You need to create the CloudFormation stack only once per account, even when installing the integration in multiple regions. If you are setting up the integration in a second region, verify that the externalId in the form matches the one for the existing CloudFormation stack on the "Parameters" tab.
+You need to create the CloudFormation stack only once per account, even when installing the integration in multiple regions. If you are setting up the integration in a second region, copy the externalId from the existing CloudFormation stack on the "Parameters" tab into the form shown above.
 
 </Alert>
```
Comment (PeloWriter) [inline src/docs/product/integrations/aws-lambda/index.mdx:49]: It's either an Alert or a Note (our old format combined them both into a muddle). From our Tips for Contributors:
## Warning, Note, Important

- **Prerequisites:** use these to list things that are required to finish. Place them prior to the procedure/process that follows so a user doesn't start and stop.
- **Note:** use these for useful context, information that should be visually called out from the core content. Use sparingly.
- **Important:** use these for information that's critical to know (this version introduces breaking changes). Use infrequently.
- **Warning:** use these for items that MUST be well understood before proceeding. These should be used very rarely.

If you determine it's a Note, use the "Note /Note" format (I can't put the full markdown here as GH is interpreting it, but enclose them with the arrow brackets)

File: src/docs/product/integrations/aws-lambda/index.mdx [hunk 3]
```diff
@@ -26,25 +26,29 @@ Connect Sentry to your AWS account to automatically add Sentry error and perform
 
    ![](project.png)
 
-1. Click on **Add Sentry's CloudFormation to your AWS**.
+1. Click on **Go To AWS**.
```
Comment (PeloWriter) [inline src/docs/product/integrations/aws-lambda/index.mdx:29]: So we can just tell people to Click, no need to Click on (fewer words increases readability)

File: src/docs/product/integrations/aws-lambda/index.mdx (hunk 2 above)
Comment (scefali) [reply src/docs/product/integrations/aws-lambda/index.mdx:49]: @PeloWriter so you mean use thee `Note` component instead of the `Alert` component?

**Pull request merged**